Makefile

```diff
@@ -91,15 +88,13 @@ image/build/master:
 	operator-sdk build quay.io/${QUAY_ORG}/${QUAY_IMAGE}:master
 
 .PHONY: image/push/master
-image/push/master:
+image/push/master: image/build/master
```

Comment:
should we not use the Jenkins as it was done in the UPS oper

Reply:
Yes, I'll remove this. When that stuff was added originally (and I think even when this PR was opened originally) the Jenkins build wasn't configured in this project, but it is now so I'll update this.

Reply:
@grdryn the Jenkins setup + CI + follow the same standard applied in the UPS oper for all is already done here: https://github.com/aerogear/app-metrics-operator/pull/11. Please, could you check and merge it instead of re-duo this job?